src/blocks/net.rs

```diff
@@ -991,13 +991,18 @@ fn get_iw_ssid(dev: &NetworkDevice) -> Result<Option<String>> {
     }
 
     let raw = raw.unwrap();
-    let result = raw
-        .stdout
-        .split(|c| *c == b'\n')
-        .filter_map(|x| IW_SSID_REGEX.find(x))
-        .next();
+    let bytes = raw.stdout.split(|c| *c == b'\n').next().unwrap();
+    let cap = IW_SSID_REGEX.captures(bytes);
+    if cap.is_none() {
```

Comment:
Isn't this case handled by `maybe_ssid_convert`?

Reply:
see below